<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBookClass</class>
 <widget class="QMainWindow" name="PhoneBookClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Запись</string>
    </property>
    <addaction name="actionAdd_New"/>
    <addaction name="actionRemove"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuRecord"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAdd_New">
   <property name="text">
    <string>Добавить запись...</string>
   </property>
   <property name="toolTip">
    <string>Добавить запись...</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Удалить запись</string>
   </property>
   <property name="toolTip">
    <string>Удалить запись</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Редактировать...</string>
   </property>
   <property name="toolTip">
    <string>Редактировать...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О Программе...</string>
   </property>
   <property name="toolTip">
    <string>О Программе...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PhoneBook.qrc"/>
 </resources>
 <connections/>
</ui>
